libc/socket/unix-socket.c

```diff
@@ -930,6 +931,14 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
+	/* Custom value (ms) added to timeout in assert since blocking interval
+	 * significantly overruns this timeout due to big kernel scheduling delays on some targets.
+	 * Typical for emulated targets.
+	 */
+	if (argc == 2) {
+		pollTimeoutDiff = atoi(argv[1]);
+	}
+
 	/* Assume /tmp dir is missing */
```

Comment:
is this comment valid? the code looks like we assume initially that it's not missing? Also fix check `if (isMissing)` to be MISRA-compliant (or use `bool`)

Reply:
The suggestion hasn't been fully taken into account. Still the comment seems to be wrong. I'd remove the comment and rename the variable name to make readable what is happening in here (`createTmpIfMissing` doesn't check the arg, it check's whether dir exists anyway and then sets the value).